phpunit.xml.dist

```diff
@@ -67,7 +67,7 @@
         </testsuite>
     </testsuites>
     <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0&amp;max[indirect]=21" />
```

Comment:
Do you think we could use a more specific approach for the target cases, instead of the number of occurrences?
I mean, something like a [baseline](https://symfony.com/doc/current/components/phpunit_bridge.html#baseline-deprecations).